### PR TITLE
Fix puppeteer vulnerability

### PR DESCRIPTION
**Pull request by HeikoTheissen**
Merge only after https://issues.chromium.org/issues/347674894 is fixed.

### CODE REVIEW

File: lib/md2html.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ filter.stdio(function ({ t, c }, format, meta) {
     level = c[0];
     return content;
   } else if (t === "HorizontalRule") {
-    var content = [{ t, c }];
+    content = [filter.HorizontalRule()];
```
Comment (ralfhandl) [inline lib/md2html.js:26]: Why not `var` here as in the preceding if-block?

Comment (ralfhandl) [reply lib/md2html.js:26]: Then please use `let` in both places

File: docs/odata-protocol/odata-protocol.pdf
Comment (ralfhandl) [inline docs/odata-protocol/odata-protocol.pdf:1]: Why is this PDF included in the PR?

Comment (HeikoTheissen) [reply docs/odata-protocol/odata-protocol.pdf:1]: I tested (with one PDF) whether the new puppeteer version generates correct PDF links.

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**